src/features/auth/ui/SignForm.tsx

```
@@ -0,0 +1,53 @@
+      <div className="w-full max-w-[600px]">
+        <div className="border border-gray-700 rounded-[10px] shadow-2xl lg:bg-gray-600/30 lg:backdrop-blur-sm">
+          <div className="p-6">
+            <div className="flex mb-8 bg-gray-800/50 rounded-[12px] p-1.5">
+              <button
+                onClick={() => setActiveTab('login')}
+                className={`flex-1 py-3 px-6 rounded-[10px] text-sm font-medium transition-all duration-200 ${
+                  activeTab === 'login'
+                    ? 'bg-[#214d35] text-white shadow-lg'
+                    : 'text-gray-400 hover:text-white hover:bg-white/5'
+                }`}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)